packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -42,6 +42,7 @@ export class AtomicQuerySummary implements InitializableComponent {
   /**
    * Whether to display the duration of the last query execution.
    */
+  //TODO: set default to false
   @Prop() enableDuration = true;
```

Comment:
Might disturb the doc generation 🤔 
```suggestion
  @Prop() enableDuration = true;  //TODO: set default to false
```